From 844e1c5bc67eeb5de998465a86b7dc3e69d7aaac Mon Sep 17 00:00:00 2001
From: Ankit Agrawal <ankit.agarwal@intel.com>
Date: Fri, 9 Aug 2024 12:59:42 +0530
Subject: [PATCH] Removed Render script and graphics mapper hidl.

android.hardware.renderscript HIDL @1.0::IDevice can no longer be used
android.hardware.graphics.mapper @4.0 can no longer be used

Tests: Prepared EB and its booted successfully.

Tracked-On: OAM-123268
Change-Id: I12da8662139360ed5940379d1d04e38d27694db9
Signed-off-by: Ankit Agrawal <ankit.agarwal@intel.com>
---
 product.mk | 2 --
 1 file changed, 2 deletions(-)

diff --git a/product.mk b/product.mk
index dae0a8d..74d1a8a 100755
--- a/product.mk
+++ b/product.mk
@@ -34,9 +34,7 @@ PRODUCT_PACKAGES +=  \
                     android.hardware.usb-service.example \
                     camera.device@1.0-impl \
                     android.hardware.camera.provider@2.4-impl \
-                    android.hardware.graphics.mapper@4.0-impl.minigbm_intel \
                     android.hardware.graphics.allocator-service.minigbm \
-                    android.hardware.renderscript@1.0-impl \
                     android.hardware.graphics.composer3-service.intel \
                     android.hardware.identity \
                     android.hardware.identity-service.example
-- 
2.34.1

